home/ivp-planning/machines/first-steps-with-3d-object.md

```diff
@@ -32,4 +32,4 @@ The last step is to simply click anywhere in the 2D- or the [3D-panel](../user-i
 
 ![](../../../.gitbook/assets/iVP\_guide\_add\_new\_machines\_placed\_machine.jpg)
 
-From here, you can either adjust [position](selecting-and-moving-objects.md#move-objects), [rotation ](scale-and-rotate-objects.md#rotate-objects)or [scale ](scale-and-rotate-objects.md#scale-objects)of the object or [modify ](customizable-machines.md)it provided it is a [customizable object](customizable-machines.md).
+From here, you can either adjust [position](selecting-and-moving-objects.md#move-objects), [rotation ](rotate-objects.md#rotate-objects)or [scale ](rotate-objects.md#scale-objects)of the object or [modify ](customizable-machines.md)it provided it is a [customizable object](customizable-machines.md).
```

Comment:
for me the links worked, maybe some hiccups because i changed the file name in another PR / Commit or else. maybe better to sort those issues after merging both branches into main (was planning to check all references and links in one pr with the renaming of UI elements)